src/components/EntityInfo.vue

```diff
@@ -266,13 +274,40 @@ export default class EntityInfo extends Vue {
     return sessionStorage.getItem('TEMP_REG_NUMBER')
   }
 
+  /**
+   * True if the business is in limited restoration, ie, the state filing is a
+   * Limited Restoration filing or Limited Restoration Extension filing.
+   */
+  get isInLimitedRestoration (): boolean {
+    return (
+      !!this.getStateFiling && (
+        EnumUtilities.isTypeRestorationLimited(this.getStateFiling) ||
+        EnumUtilities.isTypeRestorationExtension(this.getStateFiling)
+      )
+    )
+  }
+
+  /** The limited restoration active-until date, if it exists, otherwise null. */
+  get activeUntil (): string {
+    const date = this.yyyyMmDdToDate(this.getStateFiling?.restoration?.expiry)
+    return this.dateToPacificDate(date, true)
+  }
+
+  /**
+   * True if the business is authorized to continue out, ie, the state filing is a
+   * Consent to Continuation Out filing.
+   */
+  get isAuthorizedToContinueOut (): boolean {
+    return (this.getStateFiling?.hasOwnProperty(FilingTypes.CONSENT_CONTINUATION_OUT) || false)
+  }
```

Comment:
I moved these here from the store getters file since this is the only place they are used.

Reply:
OK, I'll move it back.